aiida/calculations/transfer.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Implementation of Transfer CalcJob."""
+
+import os
+from aiida import orm
+from aiida.engine import CalcJob
+from aiida.common.datastructures import CalcInfo
+
+
+def validate_instructions(instructions, _):
```

Comment:
So, since one of the checks sees that the computer provided directly is the same as the ones of the `RemoteData` nodes, I need to set up a new computer and I was trying to look for how it is done in the `aiida_localhost` fixture but I can't find it anywhere by grepping. Where is this defined??

Reply:
https://github.com/aiidateam/aiida-core/blob/a873332b80914658c45f588c7d2ce7be79053289/aiida/manage/tests/pytest_fixtures.py#L95

Reply:
Uh, I see, thanks. Question: why is this in `aiida/manage/tests` instead of `tests/...`?

Reply:
Because in this way it is also importable by plugins to use directly in their testing frameworks.
https://github.com/aiidateam/aiida-quantumespresso/blob/2c6d02f69cf273cb5f78f6f9f39839d6397793ae/tests/conftest.py#L11